### ISSUE TITLE
Can't start Burp-rest-api with own project-file

### ISSUE THREAD

**Issue by lestatk0** [vmware/burp-rest-api#46]
i tried to start gradlew bootRun -Dproject-file=/opt/project-fil.burp
 but I got
 `` Launching the Burp with options: [--project-file=/tmp/temp-project-dir-876864466082593177/temp-project.burp, --config-file=/tmp/project-options1095584310114412893.json, --user-config-file=/tmp/user-options6426525993457471961.json] ``
Why BURP started with temp project-file?

**Comment by oucema001** [comment]
I have this issue as well. Have you been able to correct it?

**Comment by adracea** [comment]
This issue is to do with the fix applied here : #52 
Changing 
```
if (!burpEdition.equalsIgnoreCase("free") && !args.containsOption(PROJECT_FILE))
```
in BurpService.java is going to enable you to use a project file if you have burp suite pro but is going to make it not work with burp free , this is just the easy cheap fix if you only use it with burp pro :)

**Comment by adracea** [comment]
@oucema001 @lestatk0 With the addition of #63 , you can run the following : 
```
gradlew bootRun -PappArgs="['--project-file=/opt/project-fil.burp']"
```
The burp-rest-api literally expects a "--project-file" argv but the `gradlew bootRun -Dproject-file` sends it in a different way (haven't checked how but most likely just like...`-Dproject-file` which makes the rest api uncomfortable using it) .
Enjoy!
